tests/tasks/test_create_user.py

```diff
@@ -0,0 +1,146 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import pytest
+from argilla.server.contexts import accounts
+from argilla.server.database import SessionLocal
+from argilla.server.models import User, UserRole
+from argilla.tasks.create_user import create_user
+from click.testing import CliRunner
+from pydantic import ValidationError
+
+
+@pytest.fixture
+def db():
+    with SessionLocal() as db, db.begin():
+        yield db
+
+        db.query(User).delete()
+        db.commit()
+
+
+def test_create_user_passing_all_params(db):
+    username = "test-user"
+    first_name = "First"
+    role = "admin"
```

Comment:
`UserRole.admin` will interpolate correctly to string in the f string:

```suggestion
    role = UserRole.admin
```